recipe_scrapers/amazingoriental.py

```diff
@@ -0,0 +1,30 @@
+from ._abstract import AbstractScraper
+
+
+class AmazingOriental(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "amazingoriental.com"
+
+    def title(self):
+        return self.soup.find('h1').get_text()
+
+    def category(self):
+        return self.soup.select("div.recipe-info .meal-type")[0].get_text()
+
+    def total_time(self):
+        return self.soup.select("dl.prepare-time dd")[0].get_text()
+
+    def yields(self):
+        return self.soup.select(".person-amount")[0].get_text()
+
+    def ingredients(self):
+        ingredients = self.soup.select(".sidebar .ingredients-wrap ul li")
+        return [ingredient.get_text() for ingredient in ingredients]
+
+    def instructions(self):
+        instructions = self.soup.select(".main-wrap ul li")
+        return [instruction.get_text() for instruction in instructions]
```

Comment:
```suggestion
        return "\n".join(instruction.get_text() for instruction in instructions)
```
Instructions output should be a single string with `\n` separators between lines rather then a full break. The line by line output would be captured under `instructions_list` which is derived from this method.